CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Changes:
 
+-   Improvements
+    -   Changed how the file filtering functions in the table view.
```

Comment:
@shaneYeti Is this a typo?

Reply:
@KallynGowdy Which part of the statement?

Reply:
@shaneYeti Well, it doesn't seem to be a complete statement.